index.html

```diff
@@ -25,7 +25,40 @@
                 <img class="menu__icon" src="./images/Header/icons/hamburger.svg" alt="Menu icon" />
             </div>
         </header>
-    </div>
+
+        <div class="hero container">
+            <div class="hero__content">
+                <div class="hero__text">
+                <h2 class="hero__welcome">Hello</h2>
+                <h1 class="hero__title">I’m <span class="bold">Sarah Jackson</span></h1>
```

Comment:
Maybe calling the class hero__title--name or title__name would be more appropriate.